src/FsPickler/FsPickler/FsPickler.fs

```diff
@@ -31,13 +31,19 @@ type FsPickler private () =
     static member CreateXmlSerializer([<O;D(null)>]?typeConverter : ITypeNameConverter, [<O;D(null)>]?indent : bool, [<O;D(null)>] ?picklerResolver : IPicklerResolver) = 
         new XmlSerializer(?typeConverter = typeConverter, ?indent = indent, ?picklerResolver = picklerResolver)
 
-    /// Decides if given type is serializable by FsPickler
-    static member IsSerializableType<'T> () : bool = 
-        resolver().IsSerializable<'T> ()
+    /// <summary>
+    ///     Decides if given type is serializable by FsPickler
+    /// </summary>
+    /// <param name="picklerResolver">Specify a custom pickler resolver/cache for serialization. Defaults to the singleton pickler cache.</param>
+    static member IsSerializableType<'T> ([<O;D(null)>] ?picklerResolver : IPicklerResolver) : bool = 
```

Comment:
Both these methods and `GeneratePickler` are trivial wrappers around `IPicklerResolver`. Are they really needed?